### ISSUE TITLE
Accuracy drop in finetuning roBERTa

### ISSUE THREAD

**Issue by Arjunsankarlal** [huggingface/transformers#1519]
## ❓ How to achieve GLUE leaderboard acc for QQP task trained with roBERTa?

I am trying to finetune roberta-base model for Quora Question Pair task. In the [GLUE Leaderboard](url) the accuracy claimed is F1 / Accuracy is 74.3/90.2

I am training with the following command to fine tune the roberta-base model,

> CUDA_VISIBLE_DEVICES=0 python run_glue.py --data_dir /home/arjun/datasets/quora_roberta --model_type roberta --model_name_or_path /home/arjun/transformers/models/roberta --task_name qqp --output_dir /home/arjun/transformers/output_models/run-2/ --do_train --do_eval --do_lower_case --logging_steps 250 --save_steps 5000

All other params are default params.

I get the accuracy after 3 epochs as (taken from the eval_results.txt file),
acc = 0.6329982984448578
acc_and_f1 = 0.3164991492224289
f1 = 0.0

Also I ran evaluation for all the checkpoints stored, for every checkpoint I got the same accuracy which is quite confusing.

I'm I doing something wrong here? How to reproduce the accuracy mentioned in GLUE leaderboard?

Thanks in advance.
